### DISCUSSION TITLE
主题左侧边栏添加小工具时 保存的时候会显示json响应不合理

### DISCUSSION THREAD

**Discussion by cnoo123** [Tokinx/Wing#28] [讨论]
主题左侧边栏添加小工具时 保存的时候会显示json响应不合理

使用默认主题也不行，但是默认主题可以进入实时预览编辑界面，编辑小工具后可以发布成功。

**Comment by Tokinx** [comment]
这个地方不是主题的功能，是WP的功能。
你要不要重新配置一下主机环境。。。
